.github/workflows/install_centos_dependencies_build.sh

```diff
@@ -1,17 +1,24 @@
+set -e
 # Install required dependencies for CentOS systems
 yum update -y
 yum group install -y "Development Tools" 
 yum install -y epel-release 
 curl -C - -O https://cmake.org/files/v3.15/cmake-3.15.7-Linux-x86_64.tar.gz
 tar xzf cmake-3.15.7-Linux-x86_64.tar.gz
 ln -s $PWD/cmake-3.15.7-Linux-x86_64/bin/cmake /usr/bin/cmake
+yum install -y openssh-server openssh-clients
```

Comment:
@shivaahir1508 why this changes here? does it related to code coverage?

Reply:
@shivaahir1508 please create a separate PR for non-code coverage changes asap.